packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -266,7 +336,10 @@ class ToOOXMLVisitor {
      * @returns {string} OOXML string
      */
     toOOXML(ciceromark) {
-        this.traverseNodes(ciceromark, []);
+        this.traverseNodes(ciceromark, [], TRANSFORMED_NODES.document, {
+            isTraversingElseCondition: false,
+            isTraversingFalseCondition: false,
```

Comment:
I will try to explain why this is a bad solution. `toOOXML` is supposed to deal with the _entire_ ciceromark which contains multiple type of nodes. The properties passed here must be true for all the nodes inside it. However, when you pass such conditions which are only true for the optional node, it does not respect the previously stated rule. Moroever, it is highly possible that these properties are not used because the CiceroMark doesn't have an `Optional` node.

Reply:
@algomaster99 
I think that we can pass an empty object here. It will work and there won't be any need for explicitly declaring these properties at the time of calling.

Reply:
> I think that we can pass an empty object here.

You completely ignored "The properties passed here must be true for all the nodes inside it. ". Things should be abstracted as much as possible so that developers don't have to deal with such minute stuff while working on a higher level.